CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
-Please see the [official issue tracker] and wiki [HowToContribute].
+Please see the [official issue tracker], [contributing.rdoc] and wiki [HowToContribute].
```

Comment:
To be valid when reading as a plain text, it would need to be prefixed by `doc/`.

Reply:
Thank you for the suggestion! Is https://github.com/ruby/ruby/pull/4266/commits/0387454fc0e18c75df86eefa390d0d4ced403205 correct understanding?